Describe how to add an item to the gift register (#1347)
magento/merchdocs: src/marketing/gift-registries.md

1. **Invitations are sent**. A link to the gift registry is included in each invitation. If [Gift Registry Search]({% link marketing/gift-registry-search.md %}) is available in the store, customers can search for specific gift registries by name, email address, or gift registry ID.

1. **Invitation recipients place orders**. Those who receive an invitation can place an order for any item directly from the gift registry. As items are sold, Magento updates the gift registry item counts, and notifies the gift registry owner.

> ## Add an item to the Gift Register

Reviewer: ```suggestion
## Add an item to a gift registry on the storefront
```

> 1. Open the product that you want to add to the gift registry event.

Reviewer: ```suggestion
1. The customer opens the product that they want to add to the gift registry event.
```

> 1. Click <span class="btn">Add to Cart</span>.

Reviewer: ```suggestion
1. Clicks/taps <span class="btn">Add to Cart</span>.
```

> 1. Click <span class="btn">View and Edit Cart</span> on the Mini Cart.

Reviewer: ```suggestion
1. Clicks/taps <span class="btn">View and Edit Cart</span> on the mini cart.
```

> 1. On the Shopping Cart page select the event you need and click <span class="btn">Add All To Gift Registry</span>.

Reviewer: ```suggestion
1. On the Shopping Cart page, selects the event they want and clicks/taps <span class="btn">Add All To Gift Registry</span>.
```

> 1. Items will be added to the gift register of the selected event.

Reviewer: This is a result and not a step

```suggestion
   Items are added to the gift registry of the selected event.
```

Author: Done, thanks 😊
